lms/static/js/spec/main.js

```diff
@@ -181,22 +181,7 @@
             },
             'underscore': {
                 deps: ['underscore.string'],
-                exports: '_',
-                init: function(UnderscoreString) {
```

Comment:
I found that this "init" code was not being called (at least at the point where I needed _.str to be defined). Also the whole mixin paradigm is something we should move away from.